pom.xml

```diff
@@ -14,7 +14,7 @@
 
     <properties>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-        <jenkins.version>1.612</jenkins.version>
+        <jenkins.version>2.7.1</jenkins.version>
```

Comment:
Okay, good to know the reason.  Unfortunately, it means that the plugin will not be compatible with Cloudbees Jenkins Enterprise which is based on Jenkins 1.651.3.